tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsIntegrationTest.java

```diff
@@ -16,7 +16,7 @@
  */
```

Comment:
No. 

Can you, please, show me an example of similar ticket? So I can create one.

Reply:
You can create a JIRA ticket just mentioning the tests (maybe linking to the code that was disabled) and saying we should update when JBOD is available. Doesn't need to be too formal. 

You may want to target 3.7 since we want to get JBOD done by then.

Reply:
[KAFKA-15646](https://issues.apache.org/jira/browse/KAFKA-15646) created.

Reply:
I prepared  #14588  that rewrites `ReplicationQuotasTestRig` in java.